spec/inject_with_accum_and_code_block_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'danject'
+
+describe Array do
+
+  it 'can calculate sum of an array' do
+    expect((0..24).to_a.danject {|accum, item| accum+item}).to eq((0..24).to_a.inject {|accum, item| accum+item})
+  end
+
+  it 'can calculate product of an array' do
+    expect((0..24).to_a.danject {|accum, item| accum*item}).to eq(0)
```

Comment:
Surrounding space missing for operator '*'.<br>Space between { and | missing.<br>Space missing inside }.